docs/pages/ops-guide/overview.md

```diff
@@ -266,19 +266,19 @@ Configuring `ROOT` vs `MOUNT` volumes may depend on the service. Some services w
 
 ## Virtual networks
 
-The SDK allows pods to join virtual networks, with the `dcos` virtual network available by defualt. You can specify that a pod should join the virtual network by using the `networks` keyword in your YAML definition. Refer to [Developers Guide](developer-guide.md) for more information about how to define virtual networks in your service.
+The SDK allows pods to join virtual networks, with the `dcos` virtual network available by defualt. You can specify that a pod should join the virtual network by using the `networks` keyword in your YAML definition. Refer to [Developers Guide](../developer-guide.md) for more information about how to define virtual networks in your service.
```

Comment:
FWIW these are about to conflict with related changes here: https://github.com/mesosphere/dcos-commons/pull/2211/files#diff-e39adaadbdc093952734c3c8e4d19c4eR269

Reply:
Does it make sense, @nickbp, for you to just pull these tweaks into your changes?

Reply:
The other PR is switching to rendering pages in dedicated directories for pretty urls (i.e., instead of `/developer-guide.html`, it's `/developer-guide/index.html`), in order to align with what docs.mesosphere.com does. So the correct links themselves are changing as of that PR

Reply:
@nickbp I'll let your PR merge first and resolve the conflicts here.

Reply:
Done: https://github.com/mesosphere/dcos-commons/pull/2213/commits/9a2e08bc97f11d9338a153264ab1a1d5f5bf5117